File: find_closest_person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        x_diff = abs(z - x)
        y_diff = abs(z - y)
        
        if x_diff < y_diff:
            return 1
        elif y_diff < x_diff:
            return 2
        else:
            return 0

File: findClosestPerson.cpp
class Solution {
public:
    int findClosest(int x, int y, int z) {
        int xDiff = std::abs(z - x);
        int yDiff = std::abs(z - y);
        if (xDiff < yDiff){
            return 1;
        }else if (yDiff < xDiff) {
            return 2;
        }else {
            return 0;
        }
    }
};